AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLConnectionScenario1Tests.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import AWSMobileClient
+@testable import AWSAPICategoryPlugin
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSAPICategoryPluginTestCommon
+
+class GraphQLConnectionScenario1Tests: XCTestCase {
```

Comment:
Here & throughout: "scenario1" isn't meaningful without a reference. Rather than making someone look up the README, consider also including the scenario definitions in the class comments of this file.